solr/core/src/java/org/apache/solr/api/AnnotatedApi.java

```diff
@@ -101,14 +101,14 @@ public EndPoint getEndPoint() {
    *                then absence of Api-s is silently ignored.
    * @return list of discovered Api-s
    */
-  public static List<Api> getApis(Class<? extends Object> theClass , Object obj, boolean allowEmpty)  {
+  public static List<Api> getApis(Class<?> theClass , Object obj, boolean allowEmpty)  {
```

Comment:
would an interface and Class<Interface> not have better typing than Class<?>?

Reply:
Probably, but that’s a bigger design decision